README.md

```diff
@@ -1 +1,37 @@
-# project_survey_form by Andrew and Akhtar
+# milestone-project/ phase-one 
+
+this is a project to learn the basics of how to use figma and flexbox
+
+### added html and css file to Github
+### created a linter to check for error in the HTML and CSS codes
+
+### created a file that should be ignored by Github
+
+### Additional description about the project and its features.
+
+## Built With
+
+### HTML
```

Comment:
- [ ] The project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
A good README should have a description that is clear, short, and to the point. It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.  Refer to this template [here](https://github.com/microverseinc/readme-template) and customise it to your needs.